### ISSUE TITLE
Microtubule dynamics - phenotype ontologies

### ISSUE THREAD

**Issue by manulera** [pombase/fypo#4126]
Based on the following issue:
https://github.com/manulera/discussions_curation/issues/4

**Comment by ValWood** [comment]
## Proposed changes

1. Rename FYPO:0000901 to abnormal microtubule dynamics during vegetative growth. Then make four children:

   * Abnormal microtubule depolymerisation rate:
     * This can already become the parent of the terms that were specific and mentioned depolymerisation <u>rate</u>, such as FYPO:0003194.
     * I can revisit the annotations of the others, and re-assign them.
   * Abnormal microtubule polymerisation rate:
     * Same
     * Terms from my paper specific to anaphase B should come as children
   * Abnormal microtubule growth event duration:
     * Since what is measured to calculate catastrophe rates is the duration of the growth events, the phenotype should be that.
     * In addition, the term "rescue rate" is problematic, since the use of "rate" kind of implies that the rate is constant, while in reality the catastrophe rate increases with microtubule age in the absence of other proteins, and even more so in the cell with the action of molecules such as kinesins-8 (klp5/klp6).
     * Terms from my paper specific to anaphase B should come as children (klp5/klp6 phenotype)
   * Abnormal microtubule rescue:
     * A bit similar to the catastrophe, rescue cannot really be measured directly, but it's distribution (ase1D perturbs distribution of rescue) or total absence can be (cls1/peg1 thermosensitive mutant prevents rescue during anaphase B). peg1 overexpression causes superstable microtubules due to excess rescue, but it's hard to provide "direct evidence" or a measurement (Bratman and Chang 2007).

   The annotations from terms that are not specific and just say "decrease depolymerisation" should be revisited and assigned to the right term (depending on whether they affected shrinkage speed or growth event duration). Same applies to others like FYPO:0005463 (decreased spatial extent of interphase microtubule polymerization), they probably can be pinpointed to increased/decreased growth/shrinkage speed or duration. I can do this in the future.

2. Rename FYPO:0007111 to depol. rate, since the child is specific to that
3. Adding "during vegetative growth" to FYPO:0003193 and FYPO:0003225, since they are children of "FYPO:0000899", "normal microtubule cytoskeleton organization during vegetative growth "
4. Linking FYPO:0006040 to the child of FYPO:0000901 with decrease microtubule depolymerisation rate.
5. Edit the synonyms as well.

### Requested terms - MT dynamics

* For the "abnormal" microtubule polym. and depolym. rates, I don't know if the default is to already add the increased/decreased children. I guess there will be annotations for both cases, and they are typically regulated by different proteins, so probably there will not be terms without annotations.

|Term name/ id|Request|New definition|New name|Child of (if added)|
|-------|-------|-------|-------|-------|
|FYPO:0000901|Change name and definition|A cellular process phenotype observed in the vegetative growth phase of the life cycle in which microtubule dynamics are abnormal.|abnormal microtubule dynamics during vegetative growth|
|Abnormal microtubule depolymerisation rate|Addition|A cellular process phenotype observed in the vegetative growth phase of the life cycle in which the rate, or speed, of microtubule polymerization, i.e. the addition of tubulin dimers to a microtubule, is abnormal (matching FYPO:0003225)||FYPO:0000901|
|Abnormal microtubule polymerisation rate|Addition|A cellular process phenotype observed in the vegetative growth phase of the life cycle in which the rate, or speed, of microtubule depolymerization, i.e. the removal of tubulin dimers to a microtubule, is abnormal (matching FYPO:0003193)||FYPO:0000901|
|Abnormal microtubule growth event duration|Addition|A cellular process phenotype observed in the vegetative growth phase of the life cycle in which the duration of microtubule growth events is abnormal||FYPO:0000901|
|Abnormal microtubule rescue|Addition|A cellular process phenotype observed in the vegetative growth phase of the life cycle in which microtubule rescue (the transition from shrinkage to growth) is abnormal. The frequency of rescues as well as the organisation can be affected.||FYPO:0000901|
|FYPO:0007111|Change name and definition|A cellular process phenotype in which the rate, or speed, of microtubule polymerization, i.e. the addition of tubulin dimers to a microtubule, is abnormal (matching FYPO:0003225)|Abnormal microtubule depolymerization rate|

### Requested terms - My paper


### Change in synonyms

These are either for FYPO:0000901, or for the terms that were already specific and mentioned growth/shrinkage **rate**.

|Term name/ id|Existing synonym|Proposed change|Reason|
|-------|-------|-------|-------|
|FYPO:0000901|[all](https://bioportal.bioontology.org/ontologies/FYPO/?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FFYPO_0000901&jump_to_nav=true)|None|I guess they can be left like this, since they mention both growth and shrinkage|
|FYPO:0003193|normal rate of microtubule catastrophe|remove|Catastrophe, not depolymerisation.|
|FYPO:0003194|The ones that mention stability and catrastrophe|remove|Catastrophe, not depolymerisation.|
|FYPO:0003190|The ones that mention stability and catrastrophe|remove|Catastrophe, not depolymerisation.|
|FYPO:0007111|catastrophe|remove|Catastrophe, not depolymerisation.

**Comment by ValWood** [comment]
revised:
FYPO:0000901 new def
A cellular process phenotype observed in the vegetative growth phase of the life cycle in which microtubule growth (polymerization), catastrophe (the transition from growth to shrinkage), shrinkage (depolymerization), or rescue (the transition from shrinkage to growth) is abnormal.

Usually we would need to obsolete a term if we change the meaning of the definition, but here since the change is subtle, and  was encapsulated by annotations to this term, I have altered the def.

**Comment by ValWood** [comment]
I could not add an equivalence axiom for "abnormal rate"

but he child terms have them:
'has part' some 
    ('increased rate'
     and (happens_during some 'mitotic interphase')
     and (happens_during some 'single-celled organism vegetative growth phase')
     and ('inheres in' some 'microtubule polymerization'))

@nicolevasilevsky @matentzn  how do I get "abnormal rate" for this grouping term for rate phenotypes? For I request this from PATO, or somewhere else? 
 It seems odd not to be able to create an equivalence axiom for the parent went he children have them.

**Comment by matentzn** [comment]
the abnormal part goes in the EQ, not the PATO class:

```
'has part' some
('increased rate'
and (happens_during some 'mitotic interphase')
and (happens_during some 'single-celled organism vegetative growth phase')
and ('inheres in' some 'microtubule polymerization') 
and ('has modifier' some 'abnormal')
)
```

Why is it necessary to have 2 'happens during' clauses? There should be just one ideally.. Do these phases happen at the same time?

**Comment by ValWood** [comment]
>the abnormal part goes in the EQ, not the PATO class:

OK that part makes sense.
So probably I cant create a definition for a grouping term for "increased or decreased" rate?



>Why is it necessary to have 2 'happens during' clauses? There should be just one ideally.. Do these phases happen at the same time?

I have no idea why there are 2, this was an existing logical def I am copying and adjusting.
I don't really understand their construction....

Looking at this the 
and (happens_during some 'single-celled organism vegetative growth phase')
looks as though it is redundant with
and (happens_during some 'mitotic interphase')

I think what has likely happened is that the 'single-celled organism vegetative growth phase' has been retained when a more specific term and equivalence axiom has been added. We can look for logical defs with 2 happens_during and revise.
In the meantime I removed the redundant looking one from FYPO:0004615. to see what happens.

**Comment by matentzn** [comment]
> So probably I cant create a definition for a grouping term for "increased or decreased" rate?

of course, just use `rate` in the grouping term, `increased rate` in the child, then you have the grouping! Or did you mean something else?

**Comment by ValWood** [comment]
doh, got it !
Thanks.
